lib/util/accessibility-routing.js

```diff
@@ -2,5 +2,7 @@
  * Determine if an itinerary has accessibility scores
  */
 export const itineraryHasAccessibilityScores = (itinerary) => {
-  return !!itinerary.legs.find((leg) => !!leg.accessibilityScore)
+  return (
+    itinerary.accessibilityScore !== null && itinerary.accessibilityScore > -1
```

Comment:
Please add a comment regarding the -1 value.

Reply:
Added in [ab6574e](https://github.com/opentripplanner/otp-react-redux/pull/1092/commits/ab6574ee9fe335fddcb9706508ebf9933ebc74a2)!

Reply:
The comment talks about zero which is good, how about the -1?

Reply:
I missed this comment before I merged, I'm sorry! I will sneak an updated comment into another otp-rr pr